cmd/coder/urls.go

```diff
@@ -35,47 +36,52 @@ var urlAccessLevel = map[string]string{
 
 func portIsValid(port string) bool {
 	p, err := strconv.ParseUint(port, 10, 16)
-	if p < 1 {
+	if p < 1 || p > 65535 {
 		// port 0 means 'any free port', which we don't support
 		err = strconv.ErrRange
 	}
 	if err != nil {
-		fmt.Println("Invalid port")
+		flog.Error("Invalid port")
 	}
 	return err == nil
 }
 
 func accessLevelIsValid(level string) bool {
 	_, ok := urlAccessLevel[level]
 	if !ok {
-		fmt.Println("Invalid access level")
+		flog.Error("Invalid access level")
```

Comment:
Changed Printf() and Println() occurrences to use flog, as that appeared to be the desired way in @ammario previous work